wgpu-core/src/pipeline.rs

```diff
@@ -234,6 +238,13 @@ pub enum CreateRenderPipelineError {
         #[source]
         error: validation::StageError,
     },
+    #[error("Internal error in stage {stage:?}: {error}")]
+    Internal {
+        stage: wgt::ShaderStage,
+        error: String,
```

Comment:
it's unfortunate that we can't specify it with `#[source]`, but we can live with this